forest/benchmarking/tests/test_example_notebooks.py

```diff
@@ -0,0 +1,196 @@
+@pytest.mark.slow
+def test_quantum_volume_nb():
+    output_str = run_notebook('quantum_volume.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 sec
+def test_qubit_spectroscopy_cz_ramsey_nb():
+    output_str = run_notebook('qubit_spectroscopy_cz_ramsey.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 sec
+def test_qubit_spectroscopy_rabi_nb():
+    output_str = run_notebook('qubit_spectroscopy_rabi.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 sec
+def test_qubit_spectroscopy_t1_nb():
+    output_str = run_notebook('qubit_spectroscopy_t1.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 sec
+def test_qubit_spectroscopy_t2_nb():
+    output_str = run_notebook('qubit_spectroscopy_t2.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 sec
+def test_random_operators_nb():
+    output_str = run_notebook('random_operators.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~5 min; passed 2019/05/13
+@pytest.mark.slow
+def test_randomized_benchmarking_interleaved_nb():
+    output_str = run_notebook('randomized_benchmarking_interleaved.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~14 sec
+def test_randomized_benchmarking_simultaneous_nb():
+    output_str = run_notebook('randomized_benchmarking_simultaneous.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~7 min; passed 2019/05/13
+@pytest.mark.slow
+def test_randomized_benchmarking_unitarity_nb():
+    output_str = run_notebook('randomized_benchmarking_unitarity.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~3 sec
+def test_readout_fidelity_nb():
+    output_str = run_notebook('readout_fidelity.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~4 sec
+def test_ripple_adder_benchmark_nb():
+    output_str = run_notebook('ripple_adder_benchmark.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~40 sec; passed 2019/05/13
+@pytest.mark.slow
+def test_robust_phase_estimation_nb():
+    output_str = run_notebook('robust_phase_estimation.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~10 sec
+def test_state_and_process_plots_nb():
+    output_str = run_notebook('state_and_process_plots.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# ~4 sec
+def test_superoperator_tools_nb():
+    output_str = run_notebook('superoperator_tools.ipynb')
+    print(output_str)
+    assert check_for_failure_and_empty_output(output_str)
+
+
+# Notebook fails at the moment
```

Comment:
Did you ever repeat this error or should we re-incorporate the test?

Reply:
I just repeated it. Cell 10 has the error `ValueError: PyQVM can only run run-and-measure style programs: Qubit 0's last operation is a gate` :(